engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -201,7 +201,8 @@ class WorkflowActor(workflowToStart: WorkflowToStart,
                     workflowHeartbeatConfig: WorkflowHeartbeatConfig,
                     totalJobsByRootWf: AtomicInteger,
                     fileHashCacheActor: Option[ActorRef],
-                    blacklistCache: Option[BlacklistCache])
+                    blacklistCache: Option[BlacklistCache],
+                    val pathBuilderFactories: List[PathBuilderFactory] = EngineFilesystems.configuredPathBuilderFactories)
```

Comment:
TOL Personal preference:

Since in production there's no reason this wouldn't be the default, I normally prefer to make things like this a field in `WorkflowActor` and override it in a new `TestWorkflowActor extends WorkflowActor`

Reply:
Good point, we don't even have a test for MWDA that uses this